### ISSUE TITLE
not-translated:Azithromycin
Azithromycin is a broad ...

### ISSUE THREAD

**Issue by KhanBugz** [Khan/khan-i18n#63447]
Url: https://pt.khanacademy.org/science/health-and-medicine/current-issues-in-health-and-medicine/Antibiotics-and-antibiotic-resistance/a/antibiotics-an-overview

User Agent: Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36

Problem Text: 
Azithromycin
Azithromycin is a broad spectrum antibiotic which is often used to treat a wide variety of infections; everything from pneumonia to sexually transmitted diseases. 
So how does it work? 
Azithromycin prevents the bacteria from multiplying by:
Blocking the cell's ability to create proteins by attaching to ribosomes in the cell.
In short, azithromycin prevents bacteria from multiplying, making it much easier for the immune system to handle the infection.

User Description: 

Next Steps: 
If this is a translation issue, fix it. Otherwise verify what is wrong, add a comment in English describing the issue and assign to tzjames.
